gpkitmodels/GP/aircraft/tail/tail_boom.py

```diff
@@ -161,7 +160,7 @@ def setup(self, tailboom, htail, state):
 
         self.tailboomJ = hasattr(tailboom, "J")
         if self.tailboomJ:
-            constraints.append(tailboom.J >= 1e-50*units("m^4"))
+            constraints.append(tailboom.J >= 1e-10*units("m^4"))
```

Comment:
was `e-50` causing issues?

Reply:
It was showing up in Bounded since it's below our default threshold of 1e-30